arch/armv7m/Makefile

```diff
@@ -5,4 +5,12 @@
 # Author: Pawel Pisarczyk
 #
 
+ifneq (, $(findstring imxrt117x, $(TARGET_SUBFAMILY)))
+	CFLAGS += -DIMXRT117X
```

Comment:
please use spaces as indentation in Makefile (2 spaces is the most common syntax). TAB is a recipe prefix (see `.RECIPEPREFIX`)  and might lead to strange errors when including sub-makefiles in unusual places

Reply:
Right, changed.